src/lender/adapters/mkr/clerk.sol

```diff
@@ -122,12 +122,18 @@ contract Clerk is Auth, Interest {
     ERC20Like public dai;
     ERC20Like public collateral;
 
+    uint public constant WAD = 10*18;
+
     // buffer to add on top of mat to avoid cdp liquidation => default 1%
     uint public matBuffer = 0.01 * 10**27;
```

Comment:
This can actually be changed like so, right? Since the contract inherits from the Interest contract, which inherits from the Math contract, which contains this variable.
```suggestion
    uint public matBuffer = 0.01 * ONE;
```